langlib/lang.transaction/src/main/ballerina/transaction.bal

```diff
@@ -35,6 +33,10 @@ public type Info readonly & record {|
    Timestamp startTime;
 |};
 
+# Information about a transaction that does not change
+# after the transaction is started.
+public type Info readonly & InfoInternal;
```

Comment:
@manuranga it seems, we have a different issue here. Info type is referred in function parameters in function pointers (Commit and Rollback handlers). If user assigned an incompatible function pointer to these handlers, compiler gives the incompatible error, but in that error, the Internal record is exposed. Please refer to https://github.com/ballerina-platform/ballerina-lang/issues/29672